### PR TITLE
Add a basic GitHub CI workflow

### PR DESCRIPTION
**Pull request by atoomic**
view sample result from workflow run
https://github.com/atoomic/p5-Mail-Pyzor/actions/runs/756348394

### CODE REVIEW

File: .github/workflows/testsuite.yml [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+    container:
+      image: perl:${{ matrix.perl-version }}
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: perl -V
+        run: perl -V
+      - name: Install Dependencies
+        uses: perl-actions/install-with-cpm@stable
```
Comment (FGasper) [inline .github/workflows/testsuite.yml:69]: Why not just `cpanm --installdeps .`? Then no need to duplicate the list of dependencies.

**Review by FGasper — CHANGES_REQUESTED** [review]

File: .github/workflows/testsuite.yml (hunk 1 above)
Comment (atoomic) [reply .github/workflows/testsuite.yml:69]: this is exactly wha is doing the cpanfile
the `perl-actions/install-with-cpm` action also works on windows

**Comment by atoomic** [comment]
switch to Dist::Zilla as discussed

File: LICENSE [hunk 2]
```diff
@@ -187,7 +192,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright 2020 cPanel, L.L.C.
+   Copyright [yyyy] [name of copyright owner]
```
Comment (FGasper) [inline LICENSE:195]: ^^ Should this template be updated?

**Review by FGasper — APPROVED** [review]
Thank you!

File: LICENSE (hunk 2 above)
Comment (atoomic) [reply LICENSE:195]: https://metacpan.org/source/LEONT/Software-License-0.103014/lib%2FSoftware%2FLicense%2FApache_2_0.pm#L231

Comment (atoomic) [reply LICENSE:195]: issue reported as https://github.com/Perl-Toolchain-Gang/Software-License/issues/76

**Pull request merged**